blib2to3/pgen2/driver.py

```diff
@@ -25,17 +25,39 @@
 
 # Pgen imports
 from . import grammar, parse, token, tokenize, pgen
+from typing import (
+    Any,
+    Callable,
+    IO,
+    Iterable,
+    List,
+    Optional,
+    Text,
+    Tuple,
+    Union,
+    Sequence,
+)
+from logging import Logger
+from blib2to3.pytree import _Convert, NL
+from blib2to3.pgen2.grammar import Grammar
+
+Path = Union[Text, "os.PathLike"]
```

Comment:
Missing a type arg to os.PathLike. We should probably turn on the `--disallow-any-generics` option.

Reply:
Fixed that and set the flag